src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -148,14 +150,22 @@ public enum DesiredLocation {
 
   private boolean isOTF = false;
 
+  private boolean isLiningUp = false;
+
   private Command driveToPose = null;
 
+  private VisionAlignment alignmentSupplier;
+
+  private PIDController driveLineupController = new PIDController(20, 0,0);
+  private PIDController rotationController = new PIDController(20, 0, 0);
```

Comment:
We should JSON these gains at least, but having them in LoggedTunable would also be helpful for testing in shop in real time

Reply:
In test mode I'd like to have the loggedtunable for this at least. JSON for sure in all cases